src/IndicatorsDash.jsx

```diff
@@ -33,7 +32,7 @@ class IndicatorsDash extends React.Component {
     const { activeBlock } = this.props;
     this.setState({ activeBlock });
 
-    const areaId = (activeBlock && activeBlock.id) ? activeBlock.id : 'LLA 0970';
+    const areaId = (activeBlock && activeBlock.id) ? activeBlock.id : this.handleCloseModal('connError');
```

Comment:
The problem with this solution is that if there is no activeBlock id areaId will be a function and will crash in loadIndicators.

Reply:
Agree, it should be a state to handleCloseModal if the state has changed, and it should be a null value if there is not an activeBlock